<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeciationWidget</class>
 <widget class="QWidget" name="SpeciationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Species Distribution</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hl_1">
     <item>
      <widget class="QLabel" name="lbl_xaxis">
       <property name="text">
        <string>X-axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_Xaxis"/>
     </item>
     <item>
      <widget class="QLabel" name="lbl_yaxis">
       <property name="text">
        <string>Y-axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_Yaxis"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_2">
     <item>
      <widget class="QLabel" name="lbl_points">
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_NPoints">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPlotErrors">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>plot errors at</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbError">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="value">
        <double>95.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_species">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Initial Titre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Final Titre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pX?</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>L</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>0.1</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>0.1</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>H</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
